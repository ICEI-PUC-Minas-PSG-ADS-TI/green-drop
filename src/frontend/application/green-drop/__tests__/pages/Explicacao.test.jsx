import React from 'react';
import { render } from '@testing-library/react-native';
import renderer from 'react-test-renderer';
import Explicacao from '../../pages/contribuir/explicação/index.jsx';

describe('<Explicacao />', () => {
it('renderiza corretamente', () => {
render(<Explicacao />);
});

it('snapshot de Explicacao', () => {
const tree = renderer.create(<Explicacao />).toJSON();
expect(tree).toMatchSnapshot();
});
});
