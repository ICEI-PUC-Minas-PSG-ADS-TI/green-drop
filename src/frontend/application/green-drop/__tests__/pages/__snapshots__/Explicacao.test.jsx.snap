// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Explicacao /> snapshot de Explicacao 1`] = `null`;
